.github/workflows/build-multi-os.yml

```diff
@@ -16,8 +16,7 @@ name: Build
 on:
   push:
     tags:
-    - 'v*'
-    - 'test*'
+    - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
This expression does not match tags with a suffix. Consider adding `(-[\S]*)?$` or similar?

Reply:
fbs doesn't allow that so we'll have to truncate the tag somehow

Reply:
sed command looks good to me. if you want, you could add a short comment to explain that we truncate because of fbs. apart from that, I think we're good to go :)

Reply:
Yes, I will add a comment when the code is changed to cache `base.json`.